.github/workflows/duty-scheduler.yaml

```diff
@@ -0,0 +1,35 @@
+name: Duty Report
+
+on:
+  # Allows manual triggering of the workflow
+  workflow_dispatch:
+  # Runs every hour
+  schedule:
+    - cron: '0 * * * *'
+
+permissions: {}
+jobs:
+  duty-report:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: "Checkout gh-pages branch"
+        uses: actions/checkout@1af3b93b6815bc44a9784bd300feb67ff0d1eeb3 # v6.0.0
+        with:
+          ref: 'gh-pages'
+
+      - name: "Generate duty report"
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          chmod +x scripts/generate-duty-report.sh
+          ./scripts/generate-duty-report.sh ./docs/duty.html 1
```

Comment:
Where's the script?

Reply:
Under the `gh-pages` branch, where all these agentic experiments are taking place 🙈
https://github.com/coreruleset/coreruleset/tree/gh-pages